extra/2-piping.js

```diff
@@ -16,26 +16,33 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(num1, num2) {
+  let sumOfNum = num1 + num2;
+  return sumOfNum;
 }
 
-function multiply() {
-
+function multiply(num1, num2) {
+  let multiplyNum = num1 * num2;
+  return multiplyNum;
 }
 
-function format() {
-
+function format(num) {
+  let numToStr = "£" + num.toString();
+  return numToStr;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = `${format(multiply(add(startingValue, 10), 2))}`;
+//This code is quite obscure  for a developer to interpret what result it is trying bring.
```

Comment:
Yes - although this code is perfectly valid and would pass the tests, it is difficult to read and work out what the developer intended